app.js

```diff
@@ -1,23 +1,53 @@
 // Task 1 - Notes for JS Evaluation Predictions
-
+ 
 // List 3 expressions that surprised you or gave unexpected results, why?
+    // 2 + 'pizza'
+    // I didn't anticipate that the number and string would be added into the same string
+
+    // 10 % 2 === 0
+    // Anticipated the output to log as false - I thought the first value also needed to be 0 for it to be interpreted as true
 
+    // 'engineering'.indexOf('G')
+    // I was confused at what the output would even provide, and more confused about what output was actually produced. Unfamiliar with indexOf and what -1 meant in the index
 // List 2 exprsssions you felt comfortable interpreting at a glance.
+    // 9 % 3 === 0
+    // 5 >= 11
 
 // List 1 expression you are still 'shaky' on, why?
-
+    // 5 === 5.0
+    // I understand === as requiring both values to be written in the exact same form. because 5.0 is not written the same way as 5 is, I'd expect the console log to be false
```

Comment:
In JS, floats and integers are not different and JS actually stores all numbers as floats. So even if you input `5` JS sees it as `5.0`